bin/polly.py

```diff
@@ -350,7 +361,7 @@ def PositiveInt(string):
     build_dir_option
 ]
 
-if args.config and not toolchain_entry.multiconfig:
+if (args.config and not toolchain_entry.multiconfig) or args.config_all:
```

Comment:
@ruslo : I left the existing logic for --config as it was.  By default, if it is a multiconfig toolchain it does not set `-DCMAKE_BUILD_TYPE=<type>`.  In the case of `--config-all` it will set `-DCMAKE_BUILD_TYPE=<type>.